javalib/src/main/scala/java/lang/process/UnixProcess.scala

```diff
@@ -179,28 +179,28 @@ object UnixProcess {
         .map(e => s"${e.getKey()}=${e.getValue()}")
     }
 
-    /*
-     * Use vfork rather than fork to avoid copying the parent process memory to the child. It also
-     * ensures that the parent won't try to read or write to the child file descriptors before the
-     * child process has called execve. In an ideal world, we'd use posix_spawn, but it doesn't
-     * support changing the working directory of the child process or closing all of the unused
-     * parent file descriptors. Using posix_spawn would require adding an additional step in which
-     * we spawned a new process that called execve with a helper binary. This may be necessary
-     * eventually to increase portability but, for now, just use vfork, which is suppported on
-     * OSX and Linux (despite warnings about vfork's future, it seems somewhat unlikely that support
-     * will be dropped soon.
-     */
-    unistd.vfork() match {
+    unistd.fork() match {
       case -1 =>
         throw new IOException("Unable to fork process")
+
       case 0 =>
-        /*
-         * It is unsafe to directly run any code in vfork2 on top of the parent's stack without
-         * creating a new stack frame on the child. To fix this, put all of the code that needs
-         * to run on the child before execve inside of a method.
+        /* The ">" quoted section below describes an unattributed requirement
```

Comment:
I think it is much nicer now so future people do not have to worry about alignment. It would be easier though to just document what you want to say about the function in the first paragraph and then just explain the extra stuff in order without the special quoting stuff. I'm sure you can just use paragraphs and the sentence order to explain the gotchas and legacy stuff so it makes sense. That might make it easier if somebody cleans up the code to remove the extra notes as well.

Reply:
Again, thank you for the review iteration.

My current thinking is that if we are having a hard time describing the
oddness, then I should do at least a trial of the real fix:  getting
rid of the oddness so that it does not need to be explained, remembered,
fixed, fixed-again, and maintained. I am being short/cross but not
with you, with time spent on quirks.

CI will tell us if it does not like that approach it will let us know.
Of course, I/we will only know when this stuff is widely 
exercised.